### PR TITLE
Update Alabama.md

### PR DESCRIPTION
**Pull request by emarcey**

### CODE REVIEW

File: reports/Alabama.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
...
+
+During a peaceful protest, police corral protestors onto a hill. Officers snatch protestors at the edges and throw them to the ground to make arrests.
+
+tags: arrest, shove, throw, push, knee
+
+id: al-hoover-1
+
+**Links**
+
+* https://twitter.com/greg_doucette/status/1269810926863163394
```
Comment (maxspencer) [inline reports/Alabama.md:13]: I searched for the original tweet Doucette's referencing here and found it. Could be good to use that as the original source?

```suggestion
* https://twitter.com/mtvtacallya/status/1267318555604910081
```

File: reports/Alabama.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
+## Hoover
+
+### Police make a series of violent arrests | Date Unknown
```
Comment (maxspencer) [inline reports/Alabama.md:3]: The original tweet I found (see comment below) was posted 11:54 May 31st (local time in AL) so I think we could safely make this 31st May

```suggestion
### Police make a series of violent arrests | May 31st
```

File: reports/Alabama.md (hunk 1 above)
Comment (emarcey) [reply reports/Alabama.md:13]: Yeah thanks. I usually try to find Doucette's source but struggled this time

File: reports/Alabama.md (hunk 2 above)
Comment (emarcey) [reply reports/Alabama.md:3]: And thanks! The Doucette ones are a mess on dates

**Pull request merged**